centos_ci.sh

```diff
@@ -279,7 +279,7 @@ function perform_release() {
   add_release_notes;
   exit_on_failure "$?" "Failed to update release notes of Minishift v$RELEASE_VERSION. Try to manually update the release notes here - https://github.com/${REPO_OWNER}/minishift/releases/tag/v$RELEASE_VERSION."
 
-  make gen_adoc_tar IMAGE_UID=$(id -u)
+  make gen_adoc_tar
```

Comment:
+1, I thought this was already removed when we changed the image. Not use in this.